backend/api/views/box.py

```diff
@@ -133,6 +133,25 @@ def get_user_information(client, user_id):
     return info
 
 
+###
+def upload_file_redirect():
+    data = request.files.get("file")
+    file_name = request.form.get("file_name")
+    box_file = upload_file(data, file_name)
+    if box_file is not None:
+        file_id = box_file["id"]
+
+        new_data = Document(7, "Post Document Test File", file_id)
+        db.session.add(new_data)
+        db.session.commit()
+        return create_response(status=200, message="success")
+    else:
+        return create_response(status=400, message="Duplicate file name")
+
+
+###
```

Comment:
I don't think this comments this out - you're gonna want the docstring triple quote (""")